### PR TITLE
fu-312: TagsInput  -  add footer with tags selection clear, cancel and apply buttons

### PR DESCRIPTION
**Pull request by AndyKIron**

### CODE REVIEW

File: projects/fusion-ui/src/lib/components/ui-components/dropdown/multi-dropdown/multi-dropdown.component.ts [hunk 1]
```diff
@@ -149,11 +149,10 @@ export class MultiDropdownComponent extends DropdownComponent implements OnInit,
                 }
             });
         }
+        this.isInSelectAllAction();
```
Comment (danielbrazg) [inline projects/fusion-ui/src/lib/components/ui-components/dropdown/multi-dropdown/multi-dropdown.component.ts:152]: why you've changed multidropdown component?

File: projects/fusion-ui/src/lib/directives/click-outside/click-outside.directive.ts [hunk 2]
```diff
@@ -43,6 +50,16 @@ export class ClickOutsideDirective implements OnInit, OnDestroy {
         }
     }
 
+    private isClickInsideByCoordinates(event: MouseEvent): boolean {
```
Comment (danielbrazg) [inline projects/fusion-ui/src/lib/directives/click-outside/click-outside.directive.ts:53]: add comment explain what this is do

**Review by danielbrazg — CHANGES_REQUESTED** [review]

File: projects/fusion-ui/src/lib/components/ui-components/dropdown/multi-dropdown/multi-dropdown.component.ts (hunk 1 above)
Comment (AndyKIron) [reply projects/fusion-ui/src/lib/components/ui-components/dropdown/multi-dropdown/multi-dropdown.component.ts:152]: It fix for select all bug in multi-dropdown without confirmation (footer).

File: projects/fusion-ui/src/lib/directives/click-outside/click-outside.directive.ts (hunk 2 above)
Comment (AndyKIron) [reply projects/fusion-ui/src/lib/directives/click-outside/click-outside.directive.ts:60]: Added comment on @Input() definition - Line#20

File: projects/fusion-ui/src/lib/directives/click-outside/click-outside.directive.ts [hunk 3]
```diff
@@ -10,11 +10,22 @@ export class ClickOutsideDirective implements OnInit, OnDestroy {
     @Input() set clickOutsideActivate(value: boolean) {
         this.listenClickOutside$.next(value);
     }
+
+    /**
+     * In case click was on element inside the component but this element was
+     * removed from DOM (tag removed on TagsList). In this case directive
+     * checked if click was inside by click event coordinates
```
Comment (danielbrazg) [inline projects/fusion-ui/src/lib/directives/click-outside/click-outside.directive.ts:17]: please add comment when to use and not why you've added this abilty

**Review by danielbrazg — APPROVED** [review]

**Pull request merged**